sentences/sv/_common.yaml

```diff
@@ -16,7 +16,7 @@ lists:
         out: "black"
       - in: "(röd | rött)"
         out: "red"
-      - in: "orange"
+      - in: "(orange | oranget | orangt)"
```

Comment:
I'm not sure about if orange should be conjugation in written language, I guess this might be a place where the speech to text mapping will have to follow the language roles.
https://svenska.se/saol/?id=2155910&pz=3
Since we might say the word as `oranget` but the spelling would still be `orange` think this will follow up with the possibility for a sentence of:
`färga lamporna orangea i köket`
Like in this example:
http://blog.svd.se/sprak/2012/03/02/det-oranga-eller-orangea-kuvertet/

Reply:
I get your point. I was looking at Wikipedia. https://sv.wiktionary.org/wiki/orange#Adjektiv

When I looked at the code I was missing a conjugation for orange, all the others had one.

Reply:
Isn't this the correct mapping

röd - orange (Ändra färg till orange i vardagsrum)
röda - orangea (Färga lamporna orangea i vardagsrum)
rött - oranget/orangt (Sätt lamporna till oranget i vardagsrum)

Reply:
I would say that it depends, in written Swedish there is no conjugation for the word "orange" but in speech it's common that you would conjugate it. But since it would use a Speech to text engine I would expect it to create the correct written word for the word spoken as "oranget" should be written as "orange"